packages/core/__tests__/rest/__unit__/ZosmfRestClient.unit.test.ts

```diff
@@ -42,12 +42,19 @@ describe("ZosmfRestClient tests", () => {
     describe("Authentication errors", () => {
 
         it("should handle basic auth error with empty string causeErrors", () => {
-            const zosmfRestClient = new ZosmfRestClient(new Session({
+            const fakeSess = new Session({
                 hostname: "dummy",
                 type: SessConstants.AUTH_TYPE_BASIC,
                 user: "fakeUser",
                 password: "fakePass"
-            }));
+            });
+
+            // ensure that some available creds are cached
+            (AuthOrder as any).cacheCredsAndAuthOrder(fakeSess.ISession,
```

Comment:
Same as previous comment(s) (regarding the need for `as any`)
```suggestion
            AuthOrder.cacheCredsAndAuthOrder(fakeSess.ISession,
```

I won't make this `"Same as previous comment"` in every location, but note that this may apply multiple times in various test files 🙏